layouts/partials/component/image-block.html

```diff
@@ -80,7 +80,7 @@
     {{- end -}}
     {{ with $credit }}
       <figcaption
-        class="is-clearfix is-size-7 is-uppercase has-text-grey-light"
+        class="is-clearfix is-size-7 is-uppercase has-text-grey-light mt-1 w-full px-1 text-right font-sans text-xs uppercase leading-tight text-credit"
```

Comment:
See above. We can just use the tw stuff, so this doesn't need to change.

Reply:
mt-1 has different size in Bulma than Tailwind, so it ends up looking a little different if you try to write one file that works in both. For the nav I just used normal BEM and then duplicated the styles in bulma.css and tw.css.